### PR TITLE
fix: width change on page load

### PR DESCRIPTION
**Pull request by WoxeOfficial**
https://imgur.com/a/RJ9omAw

### Reproduce

1. Go to one of the [Donate](https://eggsy.xyz/donate/) or [Blog](https://eggsy.xyz/blog/) pages.
2. Throttle your network to `Slow 3G`. (Should be easier to see).
3. Refresh the page.

### CODE REVIEW

**Comment by eggsy** [comment]
Thanks! Let's keep it semantic HTML friendly and use a `<main>` tag instead.

File: src/layouts/default.vue [hunk 1]
```diff
@@ -163,9 +163,11 @@ export default Vue.extend({
     <Navbar class="pt-10" />
 
     <!-- Nuxt component -->
-    <Nuxt
+    <div
```
Comment (eggsy) [inline src/layouts/default.vue:166]: Thanks! Let's keep it semantic HTML friendly and use a `<main>` tag instead.

Comment (eggsy) [inline src/layouts/default.vue:166]: Thanks! Let's keep it semantic HTML friendly and use a `<main>` tag instead.

**Review by eggsy — CHANGES_REQUESTED** [review]

File: src/layouts/default.vue (hunk 1 above)
Comment (WoxeOfficial) [reply src/layouts/default.vue]: Changed it to main tag.

Comment (eggsy) [reply src/layouts/default.vue]: Thanks.

**Pull request merged**